spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -174,6 +175,33 @@ def _find_nonsys32_bash_exe():
     return shutil.which('bash', path=nonsys32_paths)
 
 
+def _filter_directories(dir_list, include=None, include_list=None, exclude=None, exclude_list=None):
+    # Handle inclusion lists
+    if include is not None:
+        dir_list = [f for f in dir_list if re.search(include, f) is not None]
+
+    if include_list is not None:
```

Comment:
I wonder if this should be an `elif`? Or better yet, either a function docstring or a code comment should remind us close to here of the expectation that at most one of `include`, `include_list` is used, and at most one of `exclude`, `exclude_list` is used.

Reply:
I like this suggestion. If we're enforcing mutual exclusion at the function level, it makes me want to put the exception checks there, too. But, then we lose easy access to `parser.error`... :thinking:

Reply:
Ha, you can also do both if you want: use `parser.error` in the parsing code, then use an `assert` in the function, because then it would be an internal invariant. But that's optional: your current comments are clear enough.